dev/com.ibm.ws.app.manager.springboot/resources/com/ibm/ws/app/manager/springboot/internal/resources/Messages.nlsprops

```diff
@@ -96,19 +65,9 @@ error.missing.ssl=CWWKC0258E: The application failed to start because it require
 error.missing.ssl.explanation=The application is a secure application, and it requires an SSL feature to be configured in the server.xml file.
 error.missing.ssl.useraction=Configure the transportSecurity-1.0 feature in the server.xml file.
 
-#UPDATED for SpringBoot 3.0:
-#error.missing.websocket=CWWKC0259E: The application failed to start because it requires a websocket feature to be configured in the server.xml file.
-#error.missing.websocket.explanation=The application is a Web Socket application, and it requires a Web Socket feature to be configured in the server.xml file.
-#error.missing.websocket.useraction=Configure a websocket feature in the server.xml file.
-
-error.spring2.requires.websocket1.application=CWWKC0259E: Feature websocket-1.0 or websocket-1.1 must be configured: Feature springBoot-1.5 or springBoot-2.0 is configured and the application uses Web Sockets.
-error.spring2.requires.websocket1.application.explanation=The application uses Web Sockets but no Web Socket feature is configured.
-error.spring2.requires.websocket1.application.useraction=Configure one of the specified Web Socket features.
-
-#ADDED for SpringBoot 3.0:
-error.spring3.requires.websocket2.application=CWWKC0275E: Feature websocket-2.0 must be configured: Feature springBoot-3.0 is configured and the application uses Web Sockets.
-error.spring3.requires.websocket2.application.explanation=The application uses Web Sockets but no Web Socket feature is configured.
-error.spring3.requires.websocket2.application.useraction=Configure the specified Web Socket feature.
+error.missing.websocket=CWWKC0259E: The application failed to start because it requires a websocket feature to be configured in the server.xml file.
```

Comment:
```suggestion
error.missing.websocket=CWWKC0259E: The application failed to start because it requires a WebSocket feature to be configured in the server.xml file.
```